### PR TITLE
feat(clerk-js): Update color of password reveal icon

### PR DESCRIPTION
**Pull request by panteliselef**
Also change between icons for eye to eye slash

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

https://user-images.githubusercontent.com/19269911/234621400-ed548dc1-99c4-4c46-917c-679b72e77938.mov

### CODE REVIEW

File: packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx [hunk 1]
```diff
@@ -45,7 +45,10 @@ const EyeCircle = ({ width, height, ...props }: EyeCircleProps) => {
       <Icon
         elementDescriptor={descriptors.impersonationFabIcon}
         icon={Eye}
-        size={undefined}
+        sx={t => ({
```
Comment (SokratisVidros) [inline packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:48]: ❓ Is this change intentional? Are we fixing a bug?

Comment (panteliselef) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:48]: This was intentional. I had to update the eye svg icon we had. The previous one had a predefined width, height and white color inside the svg file and it was used only in the Impersonation FAB.

Now we need the eye and eye-slash icons to match so it doesn't look weird when we mount one and unmount the other.  Then changes made in the FAB are there to ensure the new Icon has similar color and size to the icon used before.

Comment (panteliselef) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:48]: ### Before 
image

### After
image

As you seen they look similar even after the changes

Comment (desiprisg) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:48]: What about the `EyeSlash` icon?

**Review by SokratisVidros — APPROVED** [review]

File: packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx (hunk 1 above)
Comment (panteliselef) [reply packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx:48]: @desiprisg EyeSlash is already as it should

**Review by desiprisg — APPROVED** [review]
